src/client_side_reply.h

```diff
@@ -133,12 +129,18 @@ class clientReplyContext : public RefCountable, public StoreClient
     void purgeAllCached();
     void forgetHit();
     bool blockedHit() const;
+    void detailStoreLookup(const char *detail);
 
     void sendBodyTooLargeError();
     void sendPreconditionFailedError();
     void sendNotModified();
     void sendNotModifiedOrPreconditionFailedError();
 
+    /// Classification of the initial Store lookup.
+    /// This very first lookup happens without the Vary-driven key augmentation.
+    /// TODO: Exclude internal Store match bans from the "mismatch" category.
+    const char *firstStoreLookup_ = nullptr;
```

Comment:
I think this field belongs to ALE. Please feel free to move it and the corresponding detailStoreLookup() method to ALE or ask me to do it. I do not insist on this change because it can be viewed as out of scope (it was not this PR that misplaced `lookup_type`, this field's orignal spelling).